test/unit/org/apache/cassandra/utils/CassandraGenerators.java

```diff
@@ -597,6 +600,21 @@ public static Gen<Token> localPartitionerToken(LocalPartitioner partitioner)
         return rs -> partitioner.getToken(bytes.generate(rs));
     }
 
+    public static Gen<IPartitioner> localPartitioner()
+    {
+        return AbstractTypeGenerators.safeTypeGen().map(LocalPartitioner::new);
+    }
+
+    public static Gen<Token> localPartitionerToken()
```

Comment:
dead code... pulled from the accord branch and keeps this consistent with the other methods that exist